package.json

```diff
@@ -10,9 +10,21 @@
   "keywords": [],
   "author": "",
   "devDependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "^2.0.0",
```

Comment:
why are you adding so many packages? tons of these are already included in @nomicfoundation/hardhat-toolbox so theyre redundant